Resources/Textures/Objects/Devices/encryption_keys.rsi/meta.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from infinity baystation 12, https://github.com/infinitystation/Baystation12/blob/073f678cdce92edb8fcd55f9ffc9f0523bf31506/icons/obj/radio.dmi",
```

Comment:
I'd prefer if the /tg/ encryption keys were used as the sprites, these don't look very good.